### ISSUE TITLE
About FPMC

### ISSUE THREAD

**Issue by lyj1998** [slientGe/Sequential_Recommendation_Tensorflow#2]
# fmc = torch.mean(fmc, dim=-1)是否需要修改为
 fmc = torch.div(torch.sum(torch.matmul(Vil, Vli.permute(0, 2, 1)), dim=-1), item_list_len.float())
item_list_seq与pred的score是否应该除以其实际的长度而不是tf.mean
